### PR TITLE
Minor updates to repo

### PR DESCRIPTION
**Pull request by oreHGA**
Updated README,
Modified application structure,
Added support for static rendering of index.html (for some weird reason I had issues setting mine up)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 <body>
   <div id="app">
-
     <div class="container" style="margin-top: 3%; margin-left: 2%;">
+      <h1>Welcome to Face Overlay App</h1>
       <div class="col-md-6">
         <div>
           <div class="col-md-6">
-          <img src="http://res.cloudinary.com/unicodeveloper/image/upload/v1505797868/glasses.png" width="200" height="100" />
+          <img src="http://res.cloudinary.com/og-tech/image/upload/v1507385206/glasses.png" width="200" height="100" />
```
Comment (unicodeveloper) [inline public/index.html]: Lol. Why did you replace the image here with the one from your account?

File: public/index.html [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
           <span> Glasses </span>
           </div>
           <div class="col-md-6">
-          <img src="http://res.cloudinary.com/unicodeveloper/image/upload/v1505794374/oie_transparent.png" width="200" height="100" />
+          <img src="http://res.cloudinary.com/og-tech/image/upload/v1507379406/harlequinmask.jpg" width="200" height="100" />
```
Comment (unicodeveloper) [inline public/index.html]: Same as here. Why did you replace the image? This is not really necessary. 

Any reason why you changed it?

File: public/index.html [hunk 3]
```diff
@@ -66,6 +66,7 @@
           },
           maskedface: null,
           loading: '',
+          status: '',
```
Comment (unicodeveloper) [inline public/index.html]: Why did you add a status attribute here? 

You added the `status` attribute but did not use it at all.

File: server/index.js [hunk 4]
```diff
@@ -8,15 +8,21 @@ const cors = require('cors');
...
-  api_key: 'xxxxxxxx',
-  api_secret: 'xxxxxxxxx'
+  api_key: 'xxxxxxx',
+  api_secret: 'xxxxxxx'
 });
 
 // Multiparty middleware
 const multipartMiddleware = multipart();
+// to load the index.html for the base route 
+app.get('/', function (req, res) {
```
Comment (unicodeveloper) [inline server/index.js]: You don't need this at all. This is an API. We want to make it strictly API based.

An API-based backend & Vue.js Frontend. The index.html file was not intended to be rendered on the backend. It's a frontend file.

**Review by unicodeveloper — COMMENTED** [review]
Please apply changes.

**Comment by unicodeveloper** [comment]
The index.html is a frontend file. Once you run the frontend using a server like [http-serve](https://www.npmjs.com/package/http-server) it will render it well.

The API backend shouldn't render any  HTML file at all.

File: public/index.html (hunk 2 above)
Comment (oreHGA) [reply public/index.html]: the link to the image was broken here, that's why this was changed

File: public/index.html (hunk 3 above)
Comment (oreHGA) [reply public/index.html]: my bad, had it there when I was testing some stuff, will remove it

**Comment by oreHGA** [comment]
yeah, I see what you mean, thanks for the review. I'll make the changes now

File: public/index.html (hunk 1 above)
Comment (unicodeveloper) [reply public/index.html]: Here is not broken tho.

File: public/index.html (hunk 2 above)
Comment (unicodeveloper) [reply public/index.html]: Okay. Make sense.

Comment (oreHGA) [reply public/index.html]: I found the link that works `http://res.cloudinary.com/unicodeveloper/image/upload/v1505794374/harlequinmask.png` So I'll change it to that too

**Comment by oreHGA** [comment]
changes made 👍🏾

**Comment by unicodeveloper** [comment]
Thank you very much. This is great. I'll merge now.
